installer/pkg/cmd/service_catalog.go

```diff
@@ -524,9 +525,24 @@ func checkDependencies() error {
 	if len(missingCmds) > 0 {
 		return fmt.Errorf("%s commands not found in the PATH", strings.Join(missingCmds, ","))
 	}
+
+	// Also print out current account, project and zone information.
+	configs, err := gcp.GetConfigMap()
+	if err != nil {
+		return fmt.Errorf("error retrieving gcloud config: %v", err)
```

Comment:
nitpick:

When GetConfigMap fails, the error message would look like:
"error retrieving gcloud config: failed to unmarshal gcloud config: ...."
This looks a bit redundant to me.

Reply:
Yeah, the whole sc installer prints error messages like this, so I just followed the pattern. We might want to rethink about how to structure an error messages because it really depends. 

Here, you think it is redundant because it is easy for user to associate operation "unmarshal" with "retrieve", however, sometimes if the error message is: "error creating service account XXX: error enabling API YYY", then the information doesn't seem to be redundant any more.

Any ideas?

Reply:
sounds good to me. following the current style is also good (for now).